tests/test_rds/test_rds_clusters.py

```diff
@@ -210,6 +210,25 @@ def test_delete_db_cluster():
     client.describe_db_clusters()["DBClusters"].should.have.length_of(0)
 
 
+@mock_rds
+def test_delete_db_cluster_do_snapshot():
+    client = boto3.client("rds", region_name="eu-north-1")
+
+    client.create_db_cluster(
+        DBClusterIdentifier="cluster-id",
+        Engine="aurora",
+        MasterUsername="root",
+        MasterUserPassword="hunter2_",
+    )
+
+    client.delete_db_cluster(
+        DBClusterIdentifier="cluster-id", FinalDBSnapshotIdentifier="final-ss"
+    )
+    client.describe_db_clusters()["DBClusters"].should.have.length_of(0)
+    snapshots = client.describe_db_cluster_snapshots()["DBClusterSnapshots"]
+    snapshots[0]["DBClusterIdentifier"] == "cluster-id"
```

Comment:
Can we get another assertion here on the snapshot name, just to double check the `final-ss` is stored correctly?

Reply:
sure thing